src/components/filters/AttributeFilter/AttributeFilter.tsx

```diff
@@ -96,6 +95,39 @@ export class AttributeFilter extends React.PureComponent<IAttributeFilterProps>
         );
     }
 
+    private isInverted() {
+        const { filter } = this.props;
+
+        return !AFM.isPositiveAttributeFilter(filter);
+    }
+
+    private getIdentifierOrUri() {
+        const { filter } = this.props;
+        const displayForm = AFM.isPositiveAttributeFilter(filter)
+            ? filter.positiveAttributeFilter.displayForm
+            : filter.negativeAttributeFilter.displayForm;
+
+        if (AFM.isObjIdentifierQualifier(displayForm)) {
+            return { identifier: displayForm.identifier };
+        }
+
+        return { uri: displayForm.uri };
+    }
+
+    private getSelection() {
+        const { filter } = this.props;
+        const filterType = Object.keys(filter)[0];
```

Comment:
This is still a bit better IMHO

`const filterBody = this.isInverted() ? filter.positiveAttributeFilter : filter.negativeAttributeFilter;`

Reply:
Thx, updated.